gateway-ha/src/test/java/io/trino/gateway/ha/router/TestResourceGroupsManager.java

```diff
@@ -50,27 +50,35 @@ public void testCreateResourceGroup()
     {
         ResourceGroupsDetail resourceGroup = new ResourceGroupsDetail();
 
-        resourceGroup.setResourceGroupId(0L);
+        resourceGroup.setResourceGroupId(0L);   // This should be ignored. Id are generated by database (AUTO_INCREMENT) starting from 1.
```

Comment:
What do you mean .. it should be ignored? I think it is ignored right?
```suggestion
        resourceGroup.setResourceGroupId(0L);   // The value is ignored. IDs are generated by the database (AUTO_INCREMENT) starting from 1.
```